src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java

```diff
@@ -264,7 +274,7 @@ private XmlFile getConfigFile() {
     @Extension
     public static class ProvisioningListener extends CloudProvisioningListener {
 
-        private final CloudStatistics stats = CloudStatistics.get();
+        private final CloudStatistics stats = getCloudStatistics();
```

Comment:
That not a preference, that how human friendly and readable code should look like. I spent multiple hours reading this limited number of classes and feel impression like Perl.  Nested classes everywhere understandable only for author.  `CloudStatistics.get()` i think that at all doesn't follow getter/setters convention, getter doesn't answer what exactly i can get from this class. Static import allow to have clean and clear readable code. The same is about `getId` `Id` class. You may end like https://github.com/jenkinsci/ldap-plugin/blob/master/src/main/java/hudson/security/LDAPSecurityRealm.java where everything is in one class.